docker-compose.yml

```diff
@@ -5,19 +5,18 @@ services:
     build: .
     container_name: hapi-fhir-jpaserver-start
     restart: on-failure
-    ports:
-      - '8080:8080'
     environment:
-      JAVA_OPTS: '-Dhapi.properties=/usr/local/tomcat/webapps/ROOT/WEB-INF/classes/hapi-r4.properties -Denable-web=false -Durl_pattern=/fhir/*'
```

Comment:
I asked for you to remove anything that was set to the default values, but we should leave in the things that are in the upstream repo to limit our changes and make it easier to apply their updates.

Reply:
In upstream repo also JAVA_OPTS are removed.

Reply:
That's fine. What about the lines after this (i.e. the `HAPI_FHIR*` vars)?